components/quizzes/careerQuiz/EduBackground.tsx

```diff
@@ -1,45 +1,123 @@
+          What level of education have you received?
         </h1>
-        <p className="text-lg font-medium px-4 ">Fill in the blanks</p>
-        <div className="text-lg text-left space-y-2">
-          <div>
-            <h3>Institution</h3>{" "}
-            <input
-              type="text"
-              name="institution"
-              className="shadow appearance-none border border-gray-500 rounded-lg px-6"
-            ></input>
-          </div>
-          <div>
-            <h3>Degree</h3>{" "}
-            <input
-              type="text"
-              name="degree"
-              className="shadow appearance-none border border-gray-500 rounded-lg px-6"
-            ></input>
+        <p className="text-lg font-medium px-4">
+          Select your highest level of education.
+        </p>
+        <div className="text-lg space-y-4">
+          <div className="text-left">
+            <label htmlFor="education-select">Education</label>
+            <select
+              id="education-select"
+              value={selectedEducationLevel || ""}
+              onChange={handleEducationLevelChange}
+              className="shadow appearance-none border border-gray-500 rounded-lg px-12 max-w-xs"
+            >
+              {Object.values(EducationLevel).map((educationLevel) => (
+                <option key={educationLevel} value={educationLevel}>
+                  {educationLevel}
+                </option>
+              ))}
+            </select>
           </div>
+          {selectedEducationLevel &&
+            selectedEducationLevel !== EducationLevel.NA &&
+            selectedEducationLevel !== EducationLevel.HighSchoolDiploma &&
+            selectedEducationLevel !== EducationLevel.GED && (
+              <div className="text-left">
+                <label htmlFor="institution">Institution</label>{" "}
+                <input
+                  type="text"
+                  name="institution"
+                  value={institution}
+                  onChange={(e) => setInstitution(e.target.value)}
+                  className="border border-gray-500 rounded-lg  px-10"
+                ></input>
+              </div>
+            )}
+          {selectedEducationLevel &&
+            selectedEducationLevel !== EducationLevel.NA &&
+            selectedEducationLevel !== EducationLevel.HighSchoolDiploma &&
+            selectedEducationLevel !== EducationLevel.GED && (
+              <div className="text-left">
+                <label htmlFor="degree" className="font-medium">
+                  Field of study
+                </label>
+                <input
+                  type="text"
+                  name="degree"
+                  value={degree}
+                  onChange={(e) => setDegree(e.target.value)}
+                  className="shadow appearance-none border border-gray-500 rounded-lg px-10"
+                />
+              </div>
+            )}
+          {showExperienceDialog && (
```

Comment:
why is this called a dialog?

Reply:
I misunderstood what dialog meant. changed to showExperienceInput